### PR TITLE
Classification config

### PR DESCRIPTION
**Pull request by rasapala**

### CODE REVIEW

File: extras/ams_wrapper/src/config.py [hunk 1]
```diff
@@ -42,4 +42,13 @@
         },
     "config_path": "/opt/ams_models/vehicle_attributes_model.json"
     },
+    {"name": "emotionsRecognition", 
+    "class": VehicleAttributes,
```
Comment (mzegla) [inline extras/ams_wrapper/src/config.py:46]: I think this class name is a little misleading 😄

File: extras/ams_models/emotions_recognition_model.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            {
+                "output_name": "prob_emotion",
+                "classes": {
+                    "neutral": 0.0,
+                    "happy": 1.0,
+                    "sad": 2.0,
+                    "surprise": 3.0,
+                    "anger": 4.0
+                }
+            }
```
Comment (mzegla) [inline extras/ams_models/emotions_recognition_model.json:23]: Don't we also need value_index mapping here?

File: extras/ams_wrapper/src/config.py (hunk 1 above)
Comment (rasapala) [reply extras/ams_wrapper/src/config.py]: Done.

File: extras/ams_models/emotions_recognition_model.json (hunk 2 above)
Comment (rasapala) [reply extras/ams_models/emotions_recognition_model.json:23]: We do not have any complex mapping here. Just output type and indexed class names. So I do not see the need to index mapping. Maybe in more complex classification classes ? Info in config json file should be enough.

File: extras/ams_wrapper/src/api/models/classification_attributes_model.py [hunk 3]
```diff
@@ -26,29 +26,28 @@
...
+        for output_name in self.labels.keys():
             attributes = []
             highest_prob = 0.0
-            for position in outputs[output].keys():
-                class_name = outputs[output][position]
-                probability = inference_output[type_name][0,int(float(position)),0,0].item()
+
+            if output_name not in inference_output:
+                message = 'Model configuration label- {}'
+                ' not found in model output - {}'.format(output_name, inference_output)
```
Comment (mzegla) [inline extras/ams_wrapper/src/api/models/classification_attributes_model.py:40]: It's a little unclear here in my opinion. Could we change this message to something like:
"Output name from model config does not match model outputs" ?

**Pull request merged**